model/task_queue.go

```diff
@@ -114,6 +114,7 @@ var (
 	taskQueueItemProjectKey       = bsonutil.MustHaveTag(TaskQueueItem{}, "Project")
 	taskQueueItemExpDurationKey   = bsonutil.MustHaveTag(TaskQueueItem{}, "ExpectedDuration")
 	taskQueueItemPriorityKey      = bsonutil.MustHaveTag(TaskQueueItem{}, "Priority")
+	// taskQueueItemDependenciesKey  = bsonutil.MustHaveTag(TaskQueueItem{}, "Dependencies")
```

Comment:
why do we comment this out (here and below)? should we just remove it?

Reply:
Removed